<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect â€” Database Dossier</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="host">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
         <property name="placeholderText">
          <string>host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="port">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>3306</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="user">
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password (saved in plantext)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="test">
         <property name="text">
          <string>Test</string>
         </property>
         <property name="icon">
          <iconset theme="know"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
